metric-schema-markdown/src/test/java/com/palantir/metric/schema/markdown/MarkdownRendererTest.java

```diff
@@ -227,8 +239,11 @@ void testComplexTagged() {
                         + "namespace docs\n"
                         + "- `namespace.metric` (meter): metric docs\n"
                         + "  - `namespaceTag`\n"
-                        + "  - `result` values (`failure`,`success`)\n"
-                        + "  - `endpoint`: Some docs");
+                        + "  - `result`\n"
+                        + "    - `failure`\n"
+                        + "    - `success`: This is a success enum\n"
+                        + "  - `endpoint`: Some docs\n"
+                        + "  - `processing` values (`bar`,`foo`)");
```

Comment:
heh I'm not sure why we sorted these originally. I'd support removing the sort call in both code-paths, even though it will produce a bit of a diff as folks upgrade.

Reply:
Done :)